mgmt/rest/v1/rbody/task.go

```diff
@@ -217,12 +217,6 @@ func (s *ScheduledTaskEnabled) ResponseBodyType() string {
 
 func assertSchedule(s schedule.Schedule, t *AddScheduledTask) {
 	switch v := s.(type) {
-	case *schedule.SimpleSchedule:
-		t.Schedule = &core.Schedule{
-			Type:     "simple",
-			Interval: v.Interval.String(),
-		}
-		return
```

Comment:
`schedule.SimpleSchedule` has no longer available, so it has to be removed from here.